core/templates/dev/head/pages/topic-viewer-page/topic-viewer-page.constants.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
These constants should go into a skill-mastery.constants.ts file (see the above comment about creating a skill mastery service)

Reply:
Done.